packages/edit-post/src/store/reducer.js

```diff
@@ -20,6 +20,18 @@ import { PREFERENCES_DEFAULTS } from './defaults';
  */
 export const DEFAULT_ACTIVE_GENERAL_SIDEBAR = 'edit-post/document';
 
+/**
+ * Higher-order reducer creator which provides the given initial state for the
+ * original reducer.
+ *
+ * @param {*} initialState Initial state to provide to reducer.
+ *
+ * @return {Function} Higher-order reducer.
+ */
+const createWithInitialState = ( initialState ) => ( reducer ) => {
```

Comment:
This is magic but should simplify `preferences` reducer moving forward. I'm still not sure how this get applied so nicely :)